karate-tests/src/test/java/mocks/sandbox/supporting-functions.js

```diff
@@ -129,7 +127,7 @@ function isValidBearerToken (token) {
     return false
   } else if (tokenParts[0] !== 'Bearer') {
     return false
-  } else if (!isValidUUID(tokenParts[1])) {
+  } else if (!isValidAuthToken(tokenParts[1])) {
```

Comment:
Seems like isValidAuthToken and containsBearerToken do a lot of the same checks (has two parts, first part is bearer), it might be worth condensing them into one function.

Reply:
Consolidated those 2 functions as one